recipes/cudadecon/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "cudaDecon" %}
+{% set version = "0.0.10" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/scopetools/cudaDecon/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 29182331bb48d37b415d50365099314af9c824059cd6e9bc687e6d403d699e8e
+
+build:
+  number: 0
+  skip: True  # [win or osx or (cuda_compiler_version in ('None', 'undefined'))]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ compiler('cuda') }}
+    - cmake
+    - make  # [not win]
+    - ninja  # [win]
```

Comment:
If we are building on Windows, should we include a `bld.bat` or are we skipping that (in which case maybe `ninja` and the selector above can be dropped)?

Reply:
@isuruf mentioned in the gitter chat that we will be getting an automated PR in a couple days following the creation of the feedstock which will help us build the Windows version. Was just leaving `ninja` in there in anticipation of that, though if you would like it removed until that time, can do that as well.

Reply:
Removing for now